operator/pom.xml

```diff
@@ -108,7 +108,8 @@
             <configuration>
               <includeScope>runtime</includeScope>
               <outputDirectory>${project.build.directory}/lib</outputDirectory>
-              <excludeGroupIds>io.prometheus,com.google.protobuf</excludeGroupIds>
+              <!-- excludeGroupIds>io.prometheus,com.google.protobuf</excludeGroupIds -->
+              <excludeGroupIds>io.prometheus</excludeGroupIds>
```

Comment:
Why this change? Are you using protobuf somewhere? Also, if you really are then we don't need the commented out line.

Reply:
Yes. IIRC, it's needed by `GsonOffsetDateTime.java` that's used when converting the JSON payload to ConversionReviewModel. At the time, I kept the line commented to be sure that it's not causing any other issues. I have removed the commented out line.

Reply:
Fixed. I have reverted this change as it's no longer necessary.